docs/kubecon-na-23.md

```diff
@@ -0,0 +1,45 @@
+# KubeCon NA '23 Demo
```

Comment:
```suggestion
# KubeCon + CloudNativeCon NA '23 Demo
```

IIRC the CNCF prefers the full name as much as possible. In the docs, I suggest we use the full name.